### PR TITLE
about page aligned

### PR DESCRIPTION
**Pull request by amitsethii**
image

### CODE REVIEW

**Comment by amitsethii** [comment]
resolve of #70

File: about.html [hunk 1]
```diff
@@ -32,17 +32,23 @@
...
-    <section id="hero">
-        <div class="right-section">
+
+    <section>
+        <div class="about">
             <h1>About</h1>
-            <p class="subtitle">Typing Test is an open source project created by Arpit Ghura. The purpose of this community project is to collaborate with web developers from around the world. Our goal is to build a useful tool to practice and improve typing skills. If you would like to contribute to our project, click on the GitHub link at the top right of the page.</p>
+            <div class="subtitle">
+            <p> Typing Test is an open source project created by Arpit Ghura.</p>
+            <p> The purpose of this community project is to collaborate with web developers from around the world.</p>
```
Comment (arpitghura) [inline about.html:41]: I think you should not break this text into multiple paragraphs.

**Review by arpitghura — CHANGES_REQUESTED** [review]
Also, make sure that the maximum width of the `.about` div is `60vw`.

**Comment by arpitghura** [comment]
Also, if you can write more content, that would be great.

File: about.html (hunk 1 above)
Comment (amitsethii) [reply about.html:41]: ok, I will make these changes.

**Comment by amitsethii** [comment]
> Also, if you can write more content, that would be great.

sure

**Comment by arpitghura** [comment]
image

These two highlighted paragraph have different font styling. Make them same.

File: about.html [hunk 2]
```diff
@@ -32,17 +32,25 @@
...
-        <div class="right-section">
+
+        <div class="about">
             <h1>About</h1>
-            <p class="subtitle">Typing Test is an open source project created by Arpit Ghura. The purpose of this community project is to collaborate with web developers from around the world. Our goal is to build a useful tool to practice and improve typing skills. If you would like to contribute to our project, click on the GitHub link at the top right of the page.</p>
+            <div class="subtitle">
+            <p> 
+            Typing Test is an open-source project created by <i>Arpit Ghura</i>.
+            The purpose of this community project is to collaborate with web developers from around the world.
+            We aim to build a valuable tool to practice and improve typing skills. Which will help you in further coding and in many gvt exams.<br>
```
Comment (arpitghura) [inline about.html:42]: instead of `gvt` write the full form i.e., `government`

**Review by arpitghura — CHANGES_REQUESTED** [review]

File: about.html (hunk 2 above)
Comment (amitsethii) [reply about.html:42]: ok sure

**Comment by amitsethii** [comment]
image

please have a look

**Comment by arpitghura** [comment]
Good work @amitsethii

**Pull request merged**